test/parallel/test-internal-module-map-asserts.js

```diff
@@ -0,0 +1,96 @@
+// Flags: --expose-internals
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const ModuleMap = require('internal/loader/ModuleMap');
+
+// ModuleMap.get, url argument should only accept string values
+{
+  const errorReg = common.expectsError({
+    code: 'ERR_INVALID_ARG_TYPE',
+    type: TypeError,
+    message: /^The "url" argument must be of type string/
+  }, 5);
+
+  const moduleMap = new ModuleMap();
+
+  assert.throws(() => moduleMap.get({}), errorReg);
+
+  assert.throws(() => moduleMap.get([]), errorReg);
+
+  assert.throws(() => moduleMap.get(true), errorReg);
+
+  assert.throws(() => moduleMap.get(1), errorReg);
+
+  assert.throws(() => moduleMap.get(() => {}), errorReg);
+}
+
+// ModuleMap.has, url argument should only accept string values
+{
+  const errorReg = common.expectsError({
+    code: 'ERR_INVALID_ARG_TYPE',
+    type: TypeError,
+    message: /^The "url" argument must be of type string/
+  }, 5);
+
+  const moduleMap = new ModuleMap();
+
+  assert.throws(() => moduleMap.has({}), errorReg);
+
+  assert.throws(() => moduleMap.has([]), errorReg);
+
+  assert.throws(() => moduleMap.has(true), errorReg);
+
+  assert.throws(() => moduleMap.has(1), errorReg);
+
+  assert.throws(() => moduleMap.has(() => {}), errorReg);
```

Comment:
Nit: We don't really need to keep all of the assertions separated by empty lines, and we usually don't do it for single-line assertions. I think it mostly blows up the visual file size in this case.